CI failure checking type of return from numpy.split

Test against NumPy nightly wheels (`2.0.0.dev0`) is failing with
```
_________________________ test_split_codes_by_offsets __________________________
[gw0] linux -- Python 3.12.1 /opt/hostedtoolcache/Python/3.12.1/x64/bin/python

    def test_split_codes_by_offsets() -> None:
        codes = np.array([1, 2, 1, 2, 79], dtype=code_dtype)
    
        list_of_codes = arr.split_codes_by_offsets(codes, np.array([0, 5], dtype=offset_dtype))
        assert isinstance(list_of_codes, list)
        assert len(list_of_codes) == 1
        util_test.assert_code_array(list_of_codes[0], 5)
        assert_array_equal(list_of_codes[0], codes)
    
        list_of_codes = arr.split_codes_by_offsets(codes, np.array([0, 2, 5], dtype=offset_dtype))
>       assert isinstance(list_of_codes, list)
E       assert False
E        +  where False = isinstance((array([1, 2], dtype=uint8), array([ 1,  2, 79], dtype=uint8)), list)

tests/test_array.py:497: AssertionError
```
This is due to a change in return type of `numpy.split` from `list` to `tuple` (numpy/numpy#25570).

--- a/lib/contourpy/array.py
+++ b/lib/contourpy/array.py
@@ -228,7 +228,7 @@ def split_codes_by_offsets(codes: cpy.CodeArray, offsets: cpy.OffsetArray) -> li
     check_offset_array(offsets)
 
     if len(offsets) > 2:
-        return np.split(codes, offsets[1:-1])
+        return list(np.split(codes, offsets[1:-1]))
     else:
         return [codes]
 
@@ -243,7 +243,7 @@ def split_points_by_offsets(
     check_offset_array(offsets)
 
     if len(offsets) > 2:
-        return np.split(points, offsets[1:-1])
+        return list(np.split(points, offsets[1:-1]))
     else:
         return [points]
 

--- a/lib/contourpy/convert.py
+++ b/lib/contourpy/convert.py
@@ -387,7 +387,7 @@ def _convert_lines_from_ChunkCombinedCode(
                     assert codes is not None
                 split_at = np.nonzero(codes == MOVETO)[0]
                 if len(split_at) > 1:
-                    separate_lines += np.split(points, split_at[1:])
+                    separate_lines += list(np.split(points, split_at[1:]))
                 else:
                     separate_lines.append(points)
         if line_type_to == LineType.Separate:

--- a/lib/contourpy/util/mpl_util.py
+++ b/lib/contourpy/util/mpl_util.py
@@ -23,8 +23,8 @@ def filled_to_mpl_paths(filled: FillReturn, fill_type: FillType) -> list[mpath.P
         for points, codes, outer_offsets in zip(*filled):
             if points is None:
                 continue
-            points = np.split(points, outer_offsets[1:-1])
-            codes = np.split(codes, outer_offsets[1:-1])
+            points = list(np.split(points, outer_offsets[1:-1]))
+            codes = list(np.split(codes, outer_offsets[1:-1]))
             paths += [mpath.Path(p, c) for p, c in zip(points, codes)]
     elif fill_type == FillType.ChunkCombinedOffsetOffset:
         paths = []

